borg/archiver.py

```diff
@@ -121,6 +121,18 @@ def print_file_status(self, status, path):
         if self.output_list and (self.output_filter is None or status in self.output_filter):
             logger.info("%1s %s", status, remove_surrogates(path))
 
+    @staticmethod
+    def build_matcher(inclexcl_patterns, paths):
+        matcher = PatternMatcher()
+        if inclexcl_patterns:
+            matcher.add_inclexcl(inclexcl_patterns)
```

Comment:
Oops. That part was missing.
I added a  custom argparse action class now:
https://github.com/leo-b/borg/blob/7d2799107164e1c21b218077fa06dd8b115686c9/borg/helpers.py#L348-L358